examples/arp_response.rs

```diff
@@ -1,17 +1,26 @@
-mod arp;
-mod pv;
-
-use crate::arp::gen_arp_response_packet;
-use crate::pv::*;
-use pnet::datalink::{interfaces, MacAddr, NetworkInterface};
+/* ARP example */
+
+use packetvisor::pv::*;
```

Comment:
Use `packetvisor::pv` and use `pv::Nic`
Just `Nic` is confusable "what library is from?"

Reply:
Changed it